index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles.css"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Selcuk Guler's Personal Portfolio website</title>
+  </head>
+  <body>
+    <main>
+      <div id="topbar">
+        <div id="appbar">
+          <div id="timer-cont">
+            <div id="timer-text">9:41</div>
+          </div>
+          <div id="statusbar">
+          </div>
+        </div>
+        <div class="container" id="toolbar">
+          <div id="logo-placeholder"></div>
+          <div id="hamburger-cont">
+            <div id="hamburger-icon"></div>
+          </div>
+        </div>
+      </div>
+      <div id="home">
+        <div id="greeting-container">
+          <h1 class="greetings">
+            <span class="black-text">Hey there.<br>I'm Selçuk.</span>
+            <br>
+            <span class="green-text">I'm a software<br>developer</span>
+          </h1>
+        </div>
+        <div id="message-cont">
+          <span id="support-txt"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</span>
+        </div>
+        <div id="sm-icons-cont">
+          <span class="sm-icons" id="twitter"></span>
+          <span class="sm-icons" id="linkedin"></span>
+          <span class="sm-icons" id="medium"></span>
+          <span class="sm-icons" id="github"></span>
+          <span class="sm-icons" id="angellist"></span>
+        </div>
+        <div id="home-bg-img"></div>
+      </div>
```

Comment:
Please implement these changes for this section of your project :

- [ ] kindly use a `section` tag for the container of this section of your code, as it represents a generic standalone section of the document , you can check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info 😄
- [ ] kindly use a `p` tag for this paragraph on `line 35-38` for semantic reasons
- [ ] please the list of icons should be listed like this ⬇️ 
   > 
              <ul>
                 <li> <a href="#"> <span class="sm-icons" id="twitter"></span>  </a> </li> 
                   .
                   .
                   .
             </ul>